administrator-guides/livechat/livechat-queues/README.md

```diff
@@ -1,9 +1,10 @@
 # Livechat Queue Types
 
-There two types of queue types on Rocket.Chat:
+There are three types of queue types on Rocket.Chat:
 
 - [Round robin [default]](#round-robin)
 - [Guest Pool (coming soon)](#guest-pool)
+- [External Service (coming soon)](#external-service)
```

Comment:
why `(coming soon)` since it's live already?

Reply:
Fixed.